auth_session_timeout/models/res_users.py

```diff
@@ -17,28 +17,70 @@
 class ResUsers(models.Model):
     _inherit = 'res.users'
 
-    def _check_session_validity(self, db, uid, passwd):
+    def _auth_timeout_urls_get(self, session, db, uid):
```

Comment:
It could be argued that you are also changing the interfaces here, but they are private so you're somewhat in the clear. I would still recommend an adapter method though- you don't want to be the reason that someone's code breaks in production.